tests/helpers.py

```diff
@@ -260,13 +274,18 @@ def create_storage_class(
     return create_resource(**sc_data, wait=False)
 
 
-def create_pvc(sc_name, pvc_name=None, size=None, wait=True):
+def create_pvc(
```

Comment:
same as above, this is risky and you'll have to ensure the calls to this also modified